app/models/job_application.rb

```diff
@@ -0,0 +1,13 @@
+class JobApplication < ApplicationRecord
+  belongs_to :company
+  belongs_to :user
+
+  validates :position_title, presence: true
+  validates :date_applied, presence: true
+  validates :status, presence: true
+  validates :notes, presence: true
+  validates :job_description, presence: true
+  validates :application_url, presence: true
+  validates :contact_information, presence: true
+  validates :company_id, presence: true
+end
```

Comment:
Nice, this is simple and clean way to start off the Job Applications. there are no validations, but this is something we can raise in a new issue to accomplish